### PR TITLE
Missing googletest dependency for Ubuntu/Debian

### PR DESCRIPTION
**Pull request by aleksraiden**
Update README.md (missing test dependency for Ubuntu/Debian)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Kvrocks has the following key features:
 ## Building kvrocks
 
 #### requirements
-
+
```
Comment (ShooterIT) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ sudo yum install -y epel-release && sudo yum install -y git gcc gcc-c++ make sna
 
 # Ubuntu/Debian
 sudo apt update
-sudo apt-get install gcc g++ make libsnappy-dev autoconf automake libtool
+sudo apt-get install gcc g++ make libsnappy-dev autoconf automake libtool googletest googletest-tools libgmock-dev libgtest-dev google-mock
```
Comment (git-hulk) [inline README.md]: `googletest libgtest-dev` should be enough? we didn't use gmock and googletest-tools

Comment (aleksraiden) [reply README.md]: Yes, good. thanks

File: README.md (hunk 1 above)
Comment (git-hulk) [inline README.md]:
```suggestion
```

**Review by git-hulk — APPROVED** [review]

**Pull request merged**